lib/libzpool/kernel.c

```diff
@@ -1071,6 +1071,50 @@ highbit64(uint64_t i)
 	return (h);
 }
 
+/*
+ * Find lowest one bit set.
+ * Returns bit number + 1 of lowest bit that is set, otherwise returns 0.
+ * This is basically a reimplementation of ffsll(), which is GNU specific.
```

Comment:
You still have a few `ffsll()` callers which should be updated to use `lowbit64()`.